### PR TITLE
Feature support goprofile pull

### PR DESCRIPTION
**Pull request by EvanLjp**
https://github.com/alibaba/ilogtail/issues/730

### CODE REVIEW

File: pkg/helper/profile/meta.go [hunk 1]
```diff
@@ -34,32 +34,27 @@ type Stack struct {
...
-	MutexKind
-	GoRoutinesKind
-	ExceptionKind
-	UnknownKind
+var (
+	CPUType       = Type{Name: "cpu", Kind: "profile_cpu"}
+	MemType       = Type{Name: "mem", Kind: "profile_mem"}
+	MutexType     = Type{Name: "mutex", Kind: "profile_mutex"}
+	BlockType     = Type{Name: "block", Kind: "profile_block"}
+	ExceptionType = Type{Name: "block", Kind: "profile_exception"}
```
Comment (EvanLjp) [inline pkg/helper/profile/meta.go:47]: exception

File: plugins/input/goprofile/discovery_kubernetes.go [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+	ExcludeContainerLabel map[string]string
+	IncludeK8sLabel       map[string]string
+	ExcludeK8sLabel       map[string]string
+	ExternalEnvTag        map[string]string
+	ExternalK8sLabelTag   map[string]string
+	K8sNamespaceRegex     string
+	K8sPodRegex           string
+	K8sContainerRegex     string
+
+	IncludeContainerLabelRegex map[string]*regexp.Regexp
```
Comment (EvanLjp) [inline plugins/input/goprofile/discovery_kubernetes.go:31]: to inner field

File: plugins/input/goprofile/discovery_kubernetes.go [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+	ExcludeContainerLabelRegex map[string]*regexp.Regexp
+	IncludeEnvRegex            map[string]*regexp.Regexp
+	ExcludeEnvRegex            map[string]*regexp.Regexp
+	K8sFilter                  *helper.K8SFilter
+
+	// Last return of GetAllAcceptedInfoV2
+	fullList      map[string]bool
+	matchList     map[string]*helper.DockerInfoDetail
+	lastClearTime time.Time
```
Comment (EvanLjp) [inline plugins/input/goprofile/discovery_kubernetes.go:40]: remove

**Review by messixukejia — CHANGES_REQUESTED** [review]

File: docs/cn/data-pipeline/input/service-goprofile.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+|-----------|--------------------|-------------------|
+| Addresses | []Address，无默认值（必填） | Address Object 数组 |
+
+2. Address 配置
+
+| 参数     | 类型，默认值                 | 说明           |
+|--------|------------------------|--------------|
+| Host   | string，无默认值（必填）        | 实例地址         |
+| Port   | int，无默认值（必填）           | 实例端口         |
+| Labels | map[string]string，`{}` | 具体实例性能数据追加标签 |
```
Comment (messixukejia) [inline docs/cn/data-pipeline/input/service-goprofile.md:37]: Labels 增加区分度吧

File: plugins/input/goprofile/discovery_static.go [hunk 5]
```diff
@@ -0,0 +1,66 @@
+package goprofile
+
```
Comment (linrunqi08) [inline plugins/input/goprofile/discovery_static.go:2]: 这个文件是不是叫discovery_host比较好

**Review by messixukejia — APPROVED** [review]

File: plugins/input/goprofile/discovery_static.go (hunk 5 above)
Comment (EvanLjp) [reply plugins/input/goprofile/discovery_static.go:2]: 和prometheus 一致

**Pull request merged**